static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -1,5 +1,7 @@
 .creative__tracking-pixel {
     position: absolute;
+    width: 1px;
```

Comment:
Yes, there is not really anything wrong or good. It is more subjective so if you don't agree then keep it as it is. However because it was there before doesn't mean it was right. It's ok to question things.